yt_dlp/extractor/patreon.py

```diff
@@ -250,20 +250,24 @@ def _real_extract(self, url):
             v_url = url_or_none(compat_urllib_parse_unquote(
                 self._search_regex(r'(https(?:%3A%2F%2F|://)player\.vimeo\.com.+app_id(?:=|%3D)+\d+)', embed_html, 'vimeo url', fatal=False)))
             if v_url:
-                return {
-                    **info,
-                    '_type': 'url_transparent',
-                    'url': VimeoIE._smuggle_referrer(v_url, 'https://patreon.com'),
-                    'ie_key': 'Vimeo',
-                }
+                v_url = VimeoIE._smuggle_referrer(v_url, 'https://patreon.com')
+
+                if self._request_webpage(v_url, video_id, 'Checking Vimeo embed URL', fatal=False, errnote=False):
+                    return {
+                        **info,
+                        '_type': 'url_transparent',
+                        'url': v_url,
+                        'ie_key': 'Vimeo',
+                    }
```

Comment:
maybe this should be
```suggestion
                    return self.url_result(v_url, 'Vimeo', **info, url_transparent=True)
```